### DISCUSSION TITLE
Grafana 8 Alert API

### DISCUSSION THREAD

**Discussion by AkshayaSPrasad** [grafana/grafana#38790] [Alerting]
I am having a requirement where I need to fetch  all the alerting instances with IP address. I used the api url    **/api/prometheus/{Recipient}/api/v1/rules** to get all the alerts, but for those dashboards with multiple instances, instead of the expected IP, the variable name **$instance** is present. Is it possible to get the IP address of the alerting instance rather than the variable name.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
